ykman/cli/info.py

```diff
@@ -50,6 +50,10 @@ def info(ctx):
     else:
         click.echo('Firmware version: Uncertain, re-run with only one '
                    'YubiKey connected')
+
+    if dev.is_fips:
+        click.echo('FIPS capable: Yes')
```

Comment:
Nitpick: Maybe print out something like `This YubiKey is capable of FIPS mode.`, since it can never be `FIPS capable: No`'

Reply:
Sure!